src/taipy/gui/server.py

```diff
@@ -74,6 +75,17 @@ def handle_message(message) -> None:
             elif "type" in message:
                 gui._manage_message(message["type"], message)
 
+    def __is_ignored(self, file_path: str) -> bool:
+        if not hasattr(self, "_ignore_matches"):
+            ignore_file = (pathlib.Path(__main__.__file__).parent / ".taipyignore") if hasattr(__main__, "__file__") else None
```

Comment:
You might want to move `.taipyignore` filename as a constant

Reply:
a variable at least, you're right